config/initializers/okcomputer.rb

```diff
@@ -75,7 +75,7 @@ def check
 OkComputer::Registry.register 'suri_url', OkComputer::HttpCheck.new(suri_url.to_s)
 
 # dor-services-app
-about_url = Dor::Config.dor_services.url + '/v1/about'
+about_url = Settings.dor_services.url + '/v1/about'
```

Comment:
See above comment. This was the only place in the codebase where we used `Dor::Config.dor_services`, so I changed it to use `Settings.dor_services` instead which allowed me to remove the code above.